splash/lua_modules/splash.lua

```diff
@@ -223,5 +223,52 @@ function Splash:on_request(cb)
   end)
 end
 
+local Response = {}
+Response.__index = Response
+
+function Response._create(py_reply)
+    local self = {
+        headers=py_reply.headers,
+        info=py_reply.info,
+        request=py_reply.request
+    }
+
+    setmetatable(self, Response)
+    
+    -- convert har headers to something more convenient
+    _request_headers = {}
+    for name, value in pairs(py_reply.request["headers"]) do
+        _request_headers[value["name"]] = value["value"]
+    end
+    py_reply.request["headers"] = _request_headers
```

Comment:
Maybe we should provide this table for `splash:on_request` as well? It may be a separate ticket though, because for `splash:on_request` we need to make `request.headers` writeable.